admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -2,9 +2,9 @@
 Config.php Parameters
 =====================
```

Comment:
This file is autogenerated anyway. No need to adjust this. Please fix this in the config/config.sample.php in the server repo.